### PR TITLE
N°7154 - PHP 8.2: Fix class properties created dynamically 

### PR DESCRIPTION
**Pull request by accognet**
Internal

### CODE REVIEW

File: classes/emailmessage.class.inc.php [hunk 1]
```diff
@@ -41,7 +41,12 @@ class EmailMessage {
 	public $oRawEmail; // raw source email
 	public $aTos;
 	public $aCCs;
-	
+	/**
+	 * @var ormdocument
```
Comment (accognet) [inline classes/emailmessage.class.inc.php:45]:
```suggestion
	 * @var DOMDocument
```

**Comment by accognet** [comment]
Accepted during support PR review

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Discussed with @azana38 ; PR was reviewed during support review, issue was about to be handled by me, PR is ok, we can merge now.

File: classes/_mailinboxbase.class.inc.php [hunk 2]
```diff
@@ -29,6 +29,13 @@ abstract class _MailInboxBase extends cmdbAbstractObject
 	protected static $iMaxImageHeight = null;
 	protected static $aExcludeAttachments = null;
 
+	/**
+	 * @var int
+	 * Property is protected. The declaration was just made to make it compatible with PHP 8.2.
```
Comment (Molkobain) [inline classes/_mailinboxbase.class.inc.php:34]:
```suggestion
	 * @var int
```

File: classes/_mailinboxbase.class.inc.php [hunk 3]
```diff
@@ -29,6 +29,12 @@ abstract class _MailInboxBase extends cmdbAbstractObject
 	protected static $iMaxImageHeight = null;
 	protected static $aExcludeAttachments = null;
 
+	/**
+	 * @var int
+	 */
```
Comment (Molkobain) [inline classes/_mailinboxbase.class.inc.php:34]:
```suggestion
	/** @var int */
```

File: classes/emailmessage.class.inc.php [hunk 4]
```diff
@@ -41,7 +41,12 @@ class EmailMessage {
 	public $oRawEmail; // raw source email
 	public $aTos;
 	public $aCCs;
-	
+	/**
+	 * @var DOMDocument
+	 * Property is protected. The declaration was just made to make it compatible with PHP 8.2.
+	 */
```
Comment (Molkobain) [inline classes/emailmessage.class.inc.php:47]:
```suggestion
	/** @var DOMDocument */
```

**Pull request merged**